rig/place_and_route/utils.py

```diff
@@ -299,8 +294,54 @@ def build_routing_tables(routes, net_keys, omit_default_routes=True):
     -------
     {(x, y): [:py:class:`~rig.routing_table.RoutingTableEntry`, ...]
     """
+    # Build full routing tables and then remove default entries from them
```

Comment:
A deprecation warning should be stuck here (and in the comments).

See https://github.com/project-rig/rig/blob/master/rig/machine_control/machine_controller.py#L1884-L1886
and https://github.com/project-rig/rig/blob/master/tests/machine_control/test_machine_controller.py#L2302-L2309
for how to trigger (and test for) deprecation warnings.